src/Util/UseStatementGenerator.php

```diff
@@ -30,9 +28,9 @@ final class UseStatementGenerator implements \Stringable
      *
      * @param string[]|array<string, string> $classesToBeImported
```

Comment:
@OskarStark - over in symfony, do you guys use any of the #[ArrayShape] attributes instead of the param notations?

Something like https://blog.jetbrains.com/phpstorm/2020/10/phpstorm-2020-3-eap-4/#arrayshape or similar?

Reply:
Can't remember

Reply:
Thank you Wouter 😃

Reply:
Thanks Wouter! I'm getting curious about using `#ArrayShape` from PHPStorm, or "cloning" it in MakerBundle's NS so we can use it here. We have quite a few multi-dimensional arrays that I've run across when refactoring the code over the past couple of weeks - I'm having to jump back and forth to figure out what the array is supposed to look like. 

On the flip side, this could be a sign we should be using a simple object representation instead of a complex array in those cases...